### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by TonyDroidd** [LegendOfMCPE/PVPWorld#1]
update this plugin please :cry:

**Comment by LukeDarling** [comment]
What part needs to be updated?

**Comment by TonyDroidd** [comment]
All the plugin: p to work on new versions of pocketmine

**Comment by TonyDroidd** [comment]
:/

**Comment by iksaku** [comment]
Okey, I may update it, but I will re-write it... is it OK @DinoKill ?

**Comment by TonyDroidd** [comment]
yeaa!! :smiley:

**Comment by lgibson02** [comment]
Yeah, I don't really care what happens to this plugin anymore. :P I abandoned it a while ago. Do what ever the hell you want.
